project/Dependencies.scala

```diff
@@ -144,7 +144,7 @@ object Dependencies {
   val pact4sCirce =       "io.github.jbwheatley"  %% "pact4s-circe"     % pact4sV
   val okHttp =            "com.squareup.okhttp3"  % "okhttp"            % "4.11.0"
 
-  val workSpaceManagerV = "0.254.824-SNAPSHOT"
+  val workSpaceManagerV = "0.254.903-SNAPSHOT"
```

Comment:
I am just curious about the SNAPSHOT version, should we use a more official one?

Reply:
Hm yeah, will try

Reply:
…nope, apparently only SNAPSHOT versions are published in Artifactory